tests/integration/jobs/helpers/jobsRequestSender.ts

```diff
@@ -44,6 +44,6 @@ export async function resettable(id: string): Promise<supertest.Response> {
   return supertest.agent(app).post(`/jobs/${id}/resettable`);
 }
 
-export async function reset(id: string): Promise<supertest.Response> {
-  return supertest.agent(app).post(`/jobs/${id}/reset`);
+export async function reset(id: string, body: Record<string, unknown>): Promise<supertest.Response> {
```

Comment:
if that don't break the integration test then there are no integration tests for that.
and for future testing the body should be optional for testing invalid request secnario